libs/ledger-live-common/src/families/near/specs.ts

```diff
@@ -0,0 +1,132 @@
+import invariant from "invariant";
+import { BigNumber } from "bignumber.js";
+import type { Transaction, NearAccount } from "./types";
+import { getCryptoCurrencyById, parseCurrencyUnit } from "../../currencies";
+import { pickSiblings } from "../../bot/specs";
+import type { AppSpec } from "../../bot/types";
+import { DeviceModelId } from "@ledgerhq/devices";
+import { acceptTransaction } from "./speculos-deviceActions";
+
+const currency = getCryptoCurrencyById("near");
+const minimalAmount = parseCurrencyUnit(currency.units[0], "0.00001");
+const stakingFee = parseCurrencyUnit(currency.units[0], "0.002");
+const maxAccount = 3;
```

Comment:
as there are already 5 different kind of mutation, i think it's a good rule of thumb to make that number greater than 5. for instance 8.

Reply:
Addressed here: https://github.com/LedgerHQ/ledger-live/pull/1126/commits/55445d90c377bd85b635f4d28588908f41391d91